docsrc/imap/reference/manpages/systemcommands/jmap_expire.rst

```diff
@@ -0,0 +1,137 @@
+For example, the value `1d3m2s` denotes a duration of one day, three
+minutes and 2 seconds. The default unit is seconds.
+
+Options
+=======
+
+.. program:: jmap_expire
+
+.. option:: -C config-file
+
+    |cli-dash-c-text|
+
+.. option:: -E duration, --notif-expire=duration
+
+    Flags JMAP notifications in the *jmapnotificationfolder* mailbox as
+    expired if their creation time is older than *duration*.
+
+    The duration must be zero or positive. Setting this flag causes the
+    JMAP notification to not be visible anymore to JMAP methods, but preserves
+    them on the file system. This is required to allow Cyrus replica to
+    synchronize their state. Also see the **--notif-expunge** option.
+
+.. option:: -X duration, --notif-expunge=duration
+
+    Removes already previously expired JMAP notifications in the
+    *jmapnotificationfolder* mailbox if their last modification time
+    is older than *duration*.  This unlinks the notification from the file
+    system and is not reversible.
+
+    The duration must be zero or positive and should be higher than the
+    synchronization of any Cyrus replica. If both the **--notif-expire**
+    and **--notif-expunge** options are given, then the expunge
+    duration must be greater than or equal to the expire duration.
+
+.. option:: -h, --help
+
+    Show help information how to use this program.
+
+.. option:: -l --lock=duration
+
+    Lock a mailbox about *duration* seconds. Without this option,
+    each mailbox is exclusively locked as long as it takes to expire
+    its data. For large mailboxes, this may get other processes stuck
+    waiting for the mailbox. With this option, the mailbox is unlocked
+    after the given duration (it may take about one second longer).
+
+    The duration must be at least one second and at most one hour.
+
+    Any remaining work for the mailbox is left until the next
+    execution of jmap_expire.
+
+.. option:: -u username, --user=username
+
+    Only find JMAP data belonging to this user, e.g.  "someone@example.com".
+    Multiple occurrences of this option cause **jmap_expire** to operate on
+    all given user names.
+
+.. option:: -v, --verbose
+
+    Enable verbose output. Multiple occurrences of this option increase
+    the verbosity level, up to 3 times.
+
+Examples
+========
+
+.. parsed-literal::
+
+    **jmap_expire **-E** *7d* **-X** *7d*
+
+..
+
+        Expire JMAP notifications for all users where the notification
+        was created one week ago or earlier. Expunge all expired
+        notifications that were last modified one week ago or earlier.
+
+.. parsed-literal::
+
+    **jmap_expire **--notif-expire**=*1d* **--lock=2s** **-u** *me@example.com*
+
+..
+
+        Expire JMAP notifications in the *jmapnotificationfolder* mailbox
+        of user *me@example.com*. Stop and close the mailbox after at most
+        2 seconds and leave any remaining work for future runs of **jmap_expire**.
```

Comment:
Can you also update the cyrus.conf example files (in `doc/examples/cyrus_conf/`) to include an example invocation in their EVENTS sections?

I'm not sure which of the example files should have it, but as a starting point I think at least all the `normal*.conf` ones should.

Reply:
added it to `normal*.conf` and `small.conf`